jobs/build-snaps.yaml

```diff
@@ -21,7 +21,7 @@
           default: '{arch}'
       - string:
           name: build_node
-          default: 'runner-amd64-bigdisk'
+          default: 'runner-amd64'
```

Comment:
bigdisk no longer needed? How come?

Reply:
We actually got new workers instead of "upgraded" ones.  They all have 50+ gb root disks, so there's no need to make this distinction -- sync and snap jobs can run on any amd64 workers.